modules/data.atmosphere/R/download.NEONmet.R

```diff
@@ -0,0 +1,486 @@
+          csvVar <- csvData$soilTempMean
+          csvQF <- finalQF
+          csvVar[which(csvQF!=0)]=NA
+          ncdata[arrLoc] <- udunits2::ud.convert(csvVar,"celsius", "K")
+        }
+      }
+      ncdf4::ncvar_put(nc, varid = soilT.var, vals = ncdata)
+    }
+    
+    # NEON.DP1.00034 CO2 at tower top (alt NEON.DP3.00009 CO2 profile) - not yet avail, don't have variable names
+    
+  ncdf4::nc_close(nc)
+  } #For loop
+  return(invisible(results))
+} #function
+
+#Description of file name convention
+  # http://data.neonscience.org:80/api/v0/data/DP1.00098.001/HEAL/2016-05/NEON.D19.HEAL.DP1.00098.001.00000.003.000.030.RH_30min.csv?package=basic
+  #NEON denotes the organizational origin of the data product and identifies the product as operational; data collected as part of a special data collection exercise are designated by a separate, unique alphanumeric code created by the PI
+  #DOM is a three-character alphanumeric code, referring to the domain of data acquisition (D01 � D20)
+  #SITE is four-character alphanumeric code, referring to the site of data acquisition; all sites are designated by a standardized four-character alphabetic code
+  #DPL is a three-character alphanumeric code, referring to data product processing level
+  #PRNUM is a five-character numeric code, referring to the data product number (see the Data Product Catalog at http://data.neonscience.org/data-product-catalog)
+  #REV is a three-digit designation, referring to the revision number of the data product; it is also used to track �Data Product Maturity�, from Engineering �Grade/Provisional to Science-Grade (001 = initial REV, Engineering-Grade or Provisional; 101 = initial REV, Science-Grade)
+  #TERMS is a five-digit designation used in data product numbering to identify a sub-product or discrete vector of metadata. Since each download file typically contains several sub-products, this field is set to 00000 in the file name to maintain consistency with the data product numbering scheme.
+  #HOR is the Spatial Index #1 and refers to measurement locations within one horizontal plane. For example, if five surface measurements were taken, one at each of the five soil array plots, the number in the HOR field would range from 001-005.
+  #VER is the Spatial Index #2 and refers to measurement locations within one vertical plane. For example, if eight temperature measurements are collected, one at each tower level, the number in the VER field would range from 010-080.
+  #TMI is the Temporal Index; it is a three-digit designation and refers to the temporal representation, averaging period, or coverage of the data product (e.g., minute, hour, month, year, sub-hourly, day, lunar month, single instance, seasonal, annual, multi-annual)
+  #DESC is an abbreviated description of the data product 
+  
+  #VER - highest number = highest level
+  #Default take HOR 001 VER (max)
+
```

Comment:
All of these comments should probably go in the `@details` documentation block so that people can access them while viewing documentation. More info [by Hadley](http://r-pkgs.had.co.nz/man.html#roxygen-comments).